Packs/Thycotic/Integrations/Thycotic/Thycotic.yml

```diff
@@ -0,0 +1,1096 @@
+      name: groupowners
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: IsApplicationAccount
+      isArray: false
+      name: isapplicationaccount
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: isGroupOwnerUpdate
+      isArray: false
+      name: isgroupownerupdate
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether the user is locked out
+      isArray: false
+      name: isLockedout
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: Number of login failures
+      isArray: false
+      name: loginfailures
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether OATH two-factor authentication is enabled
+      isArray: false
+      name: oathtwofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: Password for update object
+      isArray: false
+      name: password
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether RADIUS two-factor authentication is enabled
+      isArray: false
+      name: radiustwofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: RADIUS username
+      isArray: false
+      name: radiususername
+      required: false
+      secret: false
+    - default: false
+      description: timeOptionId (integer)
+      isArray: false
+      name: timeoptionid
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether two-factor authentication is enabled
+      isArray: false
+      name: twofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    deprecated: false
+    description: Update a single user by ID
+    execution: false
+    name: thycotic-user-update
+    outputs:
+    - contextPath: Thycotic.User.Update
+      description: User object
+      type: String
+  - arguments:
+    - default: false
+      description: User ID
+      isArray: false
+      name: id
+      required: true
+      secret: false
+    deprecated: false
+    description: Delete user by ID
+    execution: false
+    name: thycotic-user-delete
+    outputs:
+    - contextPath: Thycotic.User.Delete
+      description: User object
+      type: String
+  - arguments:
+    - default: false
+      description: Secret ID
+      isArray: false
+      name: secret_id
+      required: true
+      secret: false
+    - default: false
+      description: Password
```

Comment:
```suggestion
      description: The new password
```

Reply:
@guykeller 
A secret object is returned that contains all of the secret fields. But the object itself is one.

Reply:
Description for input parameter "password" - changed